nrf_modem/doc/CHANGELOG.rst

```diff
@@ -9,6 +9,35 @@ Changelog
 
 All notable changes to this project are documented in this file.
 
+nrf_modem 2.1.0
+***************
+
+* Major improvements to modem tracing.
+  The application can now obtain trace data using the newly introduced :c:func:`nrf_modem_trace_get` function.
+  Traces can be processed as necessary, and freed using the :c:func:`nrf_modem_trace_processed` function.
+  The following functions have been removed from the OS glue:
+    * :c:func:`nrf_modem_os_trace_put`
+    * :c:func:`nrf_modem_os_trace_alloc`
+    * :c:func:`nrf_modem_os_trace_free`
+    * :c:func:`nrf_modem_os_trace_irq_set`
+    * :c:func:`nrf_modem_os_trace_irq_clear`
+    * :c:func:`nrf_modem_os_trace_irq_enable`
+    * :c:func:`nrf_modem_os_trace_irq_disable`
+  The following functions have been removed from the :c:file:`nrf_modem.h` file:
+    * :c:func:`nrf_modem_trace_irq_handler`
+    * :c:func:`nrf_modem_trace_processed_callback`
+* Improvements to AT filters.
+  AT filters now apply to the formatted AT command.
+  The :c:member:`paused` is added to the :c:type:`nrf_modem_at_cmd_filter` structure to pause filters whenever required.
+* Added support for modem's POFWARN related errors.
+* When called with ``NRF_MSG_WAITALL``, :c:func:`nrf_recv` will now return the number of bytes received so far in case the socket is closed, or when the TCP connection is terminated by the remote peer.
```

Comment:
```suggestion
* When called with ``NRF_MSG_WAITALL``, the :c:func:`nrf_recv` function now returns the number of bytes received so far in case the socket is closed, or when the TCP connection is terminated by the remote peer.
```

Reply:
Ok